Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm

```diff
@@ -1806,13 +1806,13 @@ static URL conformFragmentIdentifierForURL(const URL& url)
         CMTimeRange timeRange = [thisRangeValue CMTimeRangeValue];
         if (!CMTIMERANGE_IS_VALID(timeRange) || CMTIMERANGE_IS_EMPTY(timeRange))
             continue;
-        
+
```

Comment:
We usually tend to update the lines nearby changes, not in the whole file.

Reply:
Chris also noted that. I'll change my editor settings.